### PR TITLE
OSA sync image in e2e

### PR DESCRIPTION
**Pull request by mjudeikis**
Test with ci sync image, instead of the global one.

cc: @openshift/sig-azure 
PTAL @kwoodson

### CODE REVIEW

File: ci-operator/templates/cluster-launch-e2e-azure.yaml [hunk 1]
```diff
@@ -10,6 +10,8 @@ parameters:
   required: true
 - name: LOCAL_IMAGE_BIN
   required: true
+- name: IMAGE_SYNC
```
Comment (kargakis) [inline ci-operator/templates/cluster-launch-e2e-azure.yaml]: Does the ci-operator populate this automagically?

**Comment by mjudeikis** [comment]
Yes

File: ci-operator/templates/cluster-launch-e2e-azure.yaml (hunk 1 above)
Comment (mjudeikis) [reply ci-operator/templates/cluster-launch-e2e-azure.yaml]: yes. `Local_IMAGE_{component}` takes images from staging, `Image_{componets}` takes from stable IS

**Pull request merged**